run_task.py

```diff
@@ -337,17 +368,25 @@ def run(self):
 		self.workspace = folders[0]
 		self.tasks = []
 
-		project_file = OSUtils.find_file_with_pattern(self.workspace, PROJECT_FILE_NAME_PATTERN)
+		project_file = self.window.project_file_name()
```

Comment:
This looks to be a breaking change, as the [`project_file_name()`](https://www.sublimetext.com/docs/api_reference.html#sublime.Window.project_file_name) method will only work if there is an opened Sublime project file. Whereas, Run Task was designed to work even if you open the project root directory via `sublime .`. With that said, I think it would be best to use both methods, i.e. call `self.window.project_file_name()` first and fallback to `OSUtils.find_file_with_pattern(self.workspace, PROJECT_FILE_NAME_PATTERN)` in case no project file is opened.